frontend/src/scenes/insights/InsightHistoryPanel/InsightHistoryPanel.tsx

```diff
@@ -23,6 +24,7 @@ const { TabPane } = Tabs
 
 interface InsightHistoryPanelProps {
     onChange?: () => void
+    id?: string
```

Comment:
Any cases where we don't pass this in? I think this could as well be mandatory.

Reply:
Yup, agreed I think I can remove the ? here

Reply:
Ended up removing id prop altogether bc of bug Michael pointed out.